backend/open_webui/retrieval/vector/dbs/qdrant.py

```
@@ -19,6 +19,8 @@
     QDRANT_GRPC_PORT,
     QDRANT_PREFER_GRPC,
 )
+from open_webui.retrieval.vector.main import VectorItem, SearchResult, GetResult
```

Ruff (0.8.2)

Redefinition of unused `VectorItem` from line 11

Remove definition: `VectorItem`

(F811)